pytest_html/plugin.py

```diff
@@ -114,8 +112,12 @@ def __init__(self, outcome, report, self_contained,
             self.self_contained = self_contained
             self.logfile = logfile
 
+            test_index = -1
+            if outcome == 'Rerun':
+                test_index = report.rerun
+
             for extra_index, extra in enumerate(getattr(report, 'extra', []),
-                                                start=1):
+                                                start=0):
```

Comment:
Isn't this zero indexed by default? I don't think we need to specify the `start` argument at all.